loqusdb/commands/view.py

```diff
@@ -117,16 +117,40 @@ def variants(
             sv_cases = True
         nr_cases = adapter.nr_cases(snv_cases=snv_cases, sv_cases=sv_cases)
 
+    if chromosome:
+        if chromosome.startswith("chr"):
+            chromosomes = [chromosome, chromosome[3:]]
+        elif not chromosome.startswith("chr"):
+            chromosomes = [chromosome, "chr" + chromosome]
+
+    if end_chromosome:
+        if end_chromosome.startswith("chr"):
+            end_chromosomes = [end_chromosome, end_chromosome[3:]]
+        elif not chromosome.startswith("chr"):
```

Comment:
Should this be end_chromosome?
```suggestion
        elif not end_chromosome.startswith("chr"):
```

Reply:
Yupe! 😄